packages/api/helper/mailer.js

```diff
@@ -1,6 +1,8 @@
 const nodemailer = require('nodemailer');
 const smtpTransport = require('nodemailer-smtp-transport');
 const emailTemplate = require('email-templates');
+const previewEmail = require('preview-email');
+const path = require('path');
```

Comment:
Nothing else was changed in this file so do we need to import these?

Reply:
I do not believe so, I will remove it now.